src/test/java/org/cactoos/scalar/AvgOfTest.java

```diff
@@ -178,4 +178,35 @@ public void withScalars() {
         );
     }
 
+    @Test
+    public void withMaxValueCollections() {
+        MatcherAssert.assertThat(
+            new AvgOf(
+                new ListOf<>(Double.MAX_VALUE, Double.MAX_VALUE)
```

Comment:
@allanws It would be nice if you test to just one, two and three `MAX_VALUE` (same to Double, Float, Long and Integer). Also change with odd and even values (max values you can). And to negative values? Any tests?